99_post_install.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -xe
+
+source logging.sh
+source common.sh
+
+KUBECONFIG=${KUBECONFIG:-ocp/auth/kubeconfig}
+
+for node_type in master node; do
+  # Disable auto reboot hosts in order to apply several mcos at the same time
+  oc patch --type=merge --patch='{"spec":{"paused":true}}' machineconfigpool/${node_type}
+
+  # Apply machine configs
+  oc create -f assets/post-install/*-${node_type}.yaml
```

Comment:
These files are missing from this commit?

Reply:
I had one mc to create a `brext` interface but it seems the bridge shall require some vlan bridge configurations that @phoracek is investigating in #31

Reply:
There is no `assets/post-install/` in the git repository or this commit

Reply:
There was a brext file but I removed it because of https://github.com/openshift-kni/install-scripts/pull/28#issuecomment-521202455
I've added a README instead.